portal/patient/_machine_config.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * @package Patient
+ *
+ * MACHINE-SPECIFIC CONFIGURATION SETTINGS
+ *
+ * The configuration settings in this file can be changed to suit the
+ * machine on which the app is running (ex. local, staging or production).
+ *
+ * This file should not be added to version control, rather a template
+ * file should be added instead and then copied for each install
+ *
+ * From phreeze package
+ * @license http://www.gnu.org/copyleft/lesser.html LGPL
+ *
+ */
+/* */
+
+	session_start();
+	if ( isset($_SESSION['pid']) && isset($_SESSION['patient_portal_onsite_two']) ) {
+		$pid = $_SESSION['pid'];
+		$ignoreAuth = true;
+		 GlobalConfig::$PORTAL = true;
+		 require_once ( dirname( __FILE__ ) . "/../../interface/globals.php" );
+	}
+	else {
+		session_destroy();
+		GlobalConfig::$PORTAL = false;
+		$ignoreAuth = false;
+		$sanitize_all_escapes = true;
+		$fake_register_globals = false;
+		require_once ( dirname( __FILE__ ) . "/../../interface/globals.php" );
+		if ( ! isset($_SESSION['authUserID']) ){
+			$landingpage = "index.php";
+			header('Location: '.$landingpage);
+			exit;
+		}
+	}
+
+require_once 'verysimple/Phreeze/ConnectionSetting.php';
+require_once ( "verysimple/HTTP/RequestUtil.php" );
+
+/**
+ * database connection settings
+ */
+GlobalConfig::$CONNECTION_SETTING = new ConnectionSetting();
+GlobalConfig::$CONNECTION_SETTING->ConnectionString = $GLOBALS['host'] . ":" . $GLOBALS['port'];
+GlobalConfig::$CONNECTION_SETTING->DBName = $GLOBALS['dbase'];
+GlobalConfig::$CONNECTION_SETTING->Username = $GLOBALS['login'];
+GlobalConfig::$CONNECTION_SETTING->Password = $GLOBALS['pass'];
+GlobalConfig::$CONNECTION_SETTING->Type = "MySQLi";
+GlobalConfig::$CONNECTION_SETTING->Charset = "utf8";
```

Comment:
Should include above line based on teh following flag:
https://github.com/openemr/openemr/blob/master/sites/default/sqlconf.php#L7